src/cli.js

```diff
@@ -53,6 +53,7 @@ const meowOptions = {
     "                      (ignore \"warning\"-level)",
     "  -s, --syntax        Specify a non-standard syntax that should be used to ",
     "                      parse source stylesheets. Options: \"scss\", \"less\", \"sugarss\"",
+    "  -e, --extract       Sets extractStyleTagsFromHtml to true",
```

Comment:
The other descriptions explain the option here instead of referring to other docs, so I'd prefer a short summary here, like "Extract and lint CSS from <style> tags in HTML structures"